pkg/collector/corechecks/system/filehandles/file_handles_windows.go

```diff
@@ -30,20 +31,21 @@ func (c *fhCheck) Run() error {
 		return err
 	}
 
-	var val float64
-
-	// counter ("Process", "Handle count")
-	if c.counter == nil {
-		c.counter, err = pdhutil.GetEnglishCounterInstance("Process", "Handle Count", "_Total")
-	}
-	if c.counter != nil {
-		val, err = c.counter.GetValue()
-	}
-	if err != nil {
-		c.Warnf("file_handle.Check: Error getting process handle count: %v", err)
+	// Fetch PDH query values
+	err = c.pdhQuery.CollectQueryData()
+	if err == nil {
+		// Get values for PDH counters
+		for metricname, cset := range c.counters {
+			var val float64
+			val, err = cset.GetValue()
+			if err == nil {
+				sender.Gauge(metricname, float64(val), "", nil)
```

Comment:
Looks like 
log.Debugf("Submitting system.fs.file_handles_in_use %v", val)
is dropped not sure if accidently or not

Reply:
I dropped it because the other checks didn't have it and I'd like to be consistent.